plugins/main/public/styles/media-queries.scss

```diff
@@ -124,4 +124,10 @@
       margin-top: 10px;
     }
   }
+
+  @media only screen and (max-width: 574px) {
```

Comment:
If we are going to change the styles of native OpenSearch components we have to wrap the style with a class of our own so we are sure we don't change the behavior of the native plugins.
You can for example wrap it with `.wz-search-bar`.



![Screenshot from 2024-10-08 13-32-36](https://github.com/user-attachments/assets/bbc90ce3-ba1c-4f9d-8cb7-59f659669061)